tests/ProjectTemplatesTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Keboola\ManageApiTest;
+
+use Keboola\ManageApi\ClientException;
+
+class ProjectTemplatesTest extends ClientTestCase
+{
+    const TEST_PROJECT_TEMPLATE_STRING_ID = 'production';
+
+    public function testListProjectTemplates()
+    {
+        $templates = $this->client->getProjectTemplates();
+
+        $this->assertGreaterThan(0, count($templates));
```

Comment:
Tenhle assert bych nahradil tim, ze v tom listu musi byt ta template s id `self::TEST_PROJECT_TEMPLATE_STRING_ID` a taky ze v odpovedi musi byt to same co v `Client::getProjectTemplate()`

Reply:
trochu jsem ten test rozšířil a doplnil jak píšeš